examples/universal/z_bytes.cxx

```diff
@@ -17,6 +17,10 @@
 #include "entity.pb.h"
 #endif
 
+#if __cplusplus >= 202002L
```

Comment:
I think this is not necessary (especially given that we do not include other related stl headers) , since the corresponding header is already included by serialization.hxx.